tests/software_tests/can/transport_interface/test_python_can.py

```
@@ -3,9 +3,9 @@
 import pytest
 from mock import AsyncMock, MagicMock, Mock, call, patch
 
+from uds.addressing import AddressingType
 from uds.can import CanAddressingFormat, CanAddressingInformation
```

Ruff (0.12.2)

`uds.can.CanAddressingFormat` imported but unused

Remove unused import: `uds.can.CanAddressingFormat`

(F401)